gas_price_test.go

```diff
@@ -36,17 +36,36 @@ func TestLoadGasPrices(t *testing.T) {
 	rawPrices, err := loadGasPrices()
 	require.NoError(t, err)
 
-	fast, err := rawPrices.Fast.Float64()
-	require.NoError(t, err)
-	fastest, err := rawPrices.Fastest.Float64()
-	require.NoError(t, err)
-	safeLow, err := rawPrices.SafeLow.Float64()
-	require.NoError(t, err)
-	average, err := rawPrices.Average.Float64()
-	require.NoError(t, err)
+	assert.GreaterOrEqual(t, rawPrices.Fastest, rawPrices.Fast)
```

Comment:
I think that the difference between `assert` and `require` is that an `assert` shouldn't cause the test to fail immediately. In this case, `require` should probably be used (I believe it also has a `GreaterOrEqual` method) since this tests core functionality of the function.

Reply:
Cool -- that makes sense. My logic (which may have not been correct) was to use `require` for pre-conditions that must be met for the tests to run, but that aren't supposed to actually test business logic, and to use `assert` for the actual assertions of the test.